File: js/ksquare/fizzBuzz.js
const fizzBuzz = () => {
  const numbers = Array.from(Array(100).keys()).reverse();
  numbers.pop();

  numbers.map(number => {
    let str = "";
    if (number % 3 === 0) str += "Fizz";
    if (number % 5 === 0) str += "Buzz";
    if (str) console.log(str);
  });
};

module.exports = fizzBuzz;

File: py/js_interviews/ksquare/fizz_buzz/fizz_buzz.py
def fizz_buzz():
    for n in range(100, -1, -1):
        str = ''
        if (n % 3 == 0):
            str += 'Fizz'
        if (n % 5 == 0):
            str += 'Buzz'
        if (str):
            print(str)
